internal/services/automation/client/client.go

```diff
@@ -2,11 +2,13 @@ package client
 
 import (
 	"github.com/Azure/azure-sdk-for-go/services/preview/automation/mgmt/2020-01-13-preview/automation"
+	"github.com/hashicorp/go-azure-sdk/resource-manager/automation/2021-06-22/automationaccount"
 	"github.com/hashicorp/terraform-provider-azurerm/internal/common"
 )
 
 type Client struct {
 	AccountClient               *automation.AccountClient
+	AccountPandoraClient        *automationaccount.AutomationAccountClient
```

Comment:
we can rename the existing client here, rather than renaming this:

```suggestion
	AccountClient        *automationaccount.AutomationAccountClient
```